src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -367,6 +371,33 @@ export class TutorExerciseDashboardComponent implements OnInit {
         this.router.navigate([route]);
     }
 
+    private openManualResultDialog(result: Result) {
+        const modalRef = this.modalService.open(ProgrammingAssessmentManualResultDialogComponent, { keyboard: true, size: 'lg' });
+        modalRef.componentInstance.participationId = result.participation!.id;
+        modalRef.componentInstance.result = result;
+        modalRef.result.then(
+            () => {
+                this.loadAll();
+            },
+            () => {
+                this.loadAll();
```

Comment:
Why did you remove the subscription to `onResultModified`? I don't think we need to load everything again the user just closed the modal without actually accepting or rejecting a complaint

Reply:
You are right. I was testing it without this change and removed it. Added back now